### ISSUE TITLE
[Reflection] Zewo MapConvertible map cause EXC_BAD_ACCESS

### ISSUE THREAD

**Issue by tomohisa** [Zewo/Zewo#195]
# MapConvertible mapping cause Memory Error !?

I talked to @paulofaria on slack, but I was using MapConvertible Class to Slack API Mapping. It seems like it mapped fine but causes EXC_BAD_ACCESS error and crash program.
## Steps to reproduce

https://www.dropbox.com/s/ckc8zkixb3ih8rl/cotest.zip?dl=0

I made sample project, following code is fine when manual mapping but for Reflection mapping, it causes error when exiting do { } block.

```
import Venice
import Axis



co {
    do {
        let jsonstring = "    {    \"name\": \"Tomo\",  \"city\": \"Fukuoka, Japan\",  \"age\": 38  }"
        let buffer = Buffer(jsonstring)
        print("co routine mapping test [Manual Mapping Start]")
        let map = try JSONMapParser.parse(buffer)
        let result = User(manualMap: map)
        print("mapped: \(result.name), \(result.city), \(result.age)")

    } catch let error {
        print("Error \(error.localizedDescription)")
    }

    print("[Manual Mapping End]")

}


co {
    do {
        let jsonstring = "    {    \"name\": \"Tomo\",  \"city\": \"Fukuoka, Japan\",  \"age\": 38  }"
        let buffer = Buffer(jsonstring)
        print("co routine mapping test [Reflection Mapping Start]")
        let map = try JSONMapParser.parse(buffer)
        let result = try User(map: map)
        print("mapped: \(result.name), \(result.city), \(result.age)")

    } catch let error {
        print("Error \(error.localizedDescription)")
    }
    print("[Reflection Mapping End]")

}
```

![Error Image on Xcode](https://www.dropbox.com/s/kalxcdnqj0x2d3a/cotest_error.png?dl=1)
## Possible solutions

@robertjpayne said it could be swift problem. It could be. Or we use unsafe pointer without retaining and that causes release mapped object.

**Comment by bradhilton** [comment]
Awesome @tomohisa, I'll take a look at this.

**Comment by bradhilton** [comment]
Looks like it's actually an issue with C strings, not Venice. It's been resolved with a new `Reflection` release, `0.14.1`. @tomohisa If you update your dependencies your project should run without a `EXC_BAD_ACCESS` error.
